src/api/admin.ts

```diff
@@ -150,7 +150,7 @@ export type ListCellIdsResponse = Array<CellId>
 export type ListActiveAppsRequest = void
 export type ListActiveAppsResponse = Array<InstalledAppId>
 
-export type AppStatusFilter =  {'Active': null }| {'Inactive': null};
+export type AppStatusFilter =  'active' | 'inactive';
```

Comment:
Typescript will support nice code preview with this syntax right? No need to make it an enum structure?

Reply:
Oh yeah it will give good hints, but that's a good point also. I like the enum better.